### ISSUE TITLE
Switch serial port detecting Serial.begin(<bauds>);

### ISSUE THREAD

**Issue by eried** [eried/Arduino#4]

**Comment by Sembazuru** [comment]
After some time using ERW with auto Serial Monitor turned on, I would suggest an if type behavior:
If "Serial.begin(<baud>)" exists uncommented, then open Serial Monitor set to <baud>.

I'm not sure where to (or can) do this search; whether to scan the base .ino file or the resulting .cpp file after the preprocessing step happens.

**Comment by eried** [comment]
If "Serial.begin()" exists uncommented, then open Serial Monitor set to <?>... set to what? I don't quite understand your request

**Comment by Sembazuru** [comment]
Crap. My fault for not previewing my post before committing it. Both inside the parentheses and right before the period I had the word "baud" surrounded by angle bracket pairs. I guess it was interpreted as an unsupported tag and ignored... Let me try again with different brackets... :/

If "Serial.begin([baud])" exists uncommented, then open Serial Monitor set to [baud].
